### PR TITLE
Package Version Updates - torch 211 tf 215

### PR DESCRIPTION
**Pull request by mvamsiDO**
Updated package and library versions to be consistent with https://github.com/Paperspace/ml-in-a-box/tree/main/ubuntu-22

### CODE REVIEW

**Comment by mvamsiDO** [comment]
Testing on various GPU families on Gradient Notebooks:

| GPU/Family       | Nvidia-smi?      | PyTorch?     | Tensorflow?    | CudaNN test? | 
| -------------    | -------------    | -------------| -------------  | -------------|
| P6000            | No, DLVM         | Yes          | Yes            | Yes          |
| RTX4000          | No, DLVM         | Yes          | Yes            | Yes          |
| A6000            | No, DLVM         | Yes          | Yes            | Yes          |
| V100-32          | No, DLVM         | Yes          | Yes            | Yes          |
| A100             | No, DLVM         | Yes          | Yes            | Yes          |

`DLVM` : `Driver/library version mismatch`

**Comment by mvamsiDO** [comment]
Testing on various GPU families on Gradient Notebooks:

| GPU/Family       | Nvidia-smi?      | PyTorch?     | Tensorflow?    | CudaNN test? | 
| -------------    | -------------    | -------------| -------------  | -------------|
| P6000            | No, DLVM         | Yes          | Yes            | Yes          |
| RTX4000          | No, DLVM         | Yes          | Yes            | Yes          |
| A6000            | No, DLVM         | Yes          | Yes            | Yes          |
| V100-32          | No, DLVM         | Yes          | Yes            | Yes          |
| A100             | No, DLVM         | Yes          | Yes            | Yes          |

`DLVM` : `Driver/library version mismatch`

**Comment by mvamsiDO** [comment]
Testing on various GPU families on Gradient Notebooks: 

| GPU/Family       | Nvidia-smi?      | PyTorch?     | Tensorflow?    | CudaNN test? | 
| -------------    | -------------    | -------------| -------------  | -------------|
| P6000            | Yes              | Yes          | Yes            | Yes          |
| RTX4000          | Yes              | Yes          | Yes            | Yes          |
| A6000            | Yes              | Yes          | Yes            | Yes          |
| V100-32          | Yes              | Yes          | Yes            | Yes          |
| A100             | Yes              | Yes          | Yes            | Yes          |
| A100x2           | Yes              | Yes          | Yes            | Yes          |

- Accelerate/MultiGPU tests:

| GPU/Family       | Nvidia-smi?      | Accelerate?  | PyTorch-MultiGPU? | 
| -------------    | -------------    | -------------|-------------|
| A100             | Yes              | Yes          | NA          |
| A4000x2          | Yes              | Yes          | Yes         |
| A100x2           | Yes              | Yes          | Yes         |

**Comment by mvamsiDO** [comment]
> Testing on various GPU families on Gradient Notebooks:
> 
> GPU/Family	Nvidia-smi?	PyTorch?	Tensorflow?	CudaNN test?
> P6000	Yes	Yes	Yes	Yes
> RTX4000	Yes	Yes	Yes	Yes
> A6000	Yes	Yes	Yes	Yes
> V100-32	Yes	Yes	Yes	Yes
> A100	Yes	Yes	Yes	Yes
> * Accelerate/MultiGPU tests:
> 
> GPU/Family	Nvidia-smi?	Accelerate?	PyTorch-MultiGPU?
> A100	Yes	Yes	NA
> A4000x2	Yes	Yes	Yes

**Comment by mvamsiDO** [comment]
Ran Sanity Tests!

| GPU/Family       | Nvidia-smi?      | PyTorch?     | Tensorflow?    | CudaNN test? | 
| -------------    | -------------    | -------------| -------------  | -------------|
| A5000x2             | Yes              | Yes          | Yes            | Yes          |
| RTX4000            | Yes              | Yes          | Yes            | Yes          |
| A100x2             | Yes              | Yes          | Yes            | Yes          |


- Accelerate/Multi GPU tests:

| GPU/Family       | Nvidia-smi?      | Accelerate?  | PyTorch-MultiGPU? | 
| -------------    | -------------    | -------------|-------------|
| A5000x2          | Yes              | Yes          | Yes         |
| A100x2          | Yes              | Yes          | Yes         |


Pushed the Docker Image: [paperspace/gradient-base:pt211-tf215-cudatk120-py311-20240202](https://hub.docker.com/r/paperspace/gradient-base/tags)
